jdaviz/configs/default/plugins/subset_tools/subset_tools.py

```diff
@@ -209,6 +209,12 @@ def get_regions(self, region_type=None, list_of_subset_labels=None,
             the native data unit. If True, subsets are returned in the spectral
             axis display unit set in the Unit Conversion plugin.
 
+        return_sky_region : bool or None, optional
+            If None (default) or True, then the returned region will be ``SkyRegion`` if the
+            configuration is Imviz and the data is linked by WCS, or if the configuration
+            is Cubeviz and the data has '_orig_spatial_wcs' in the metadata. If set to False, a
```

Comment:
maybe just say 'is Cubeviz and the data has a WCS', since _orig_spatial_wcs is something we populate.